File: projects/engineering.purdue.edu/scriptingwithobjects/swocode/chap4/Join.pl
#!/usr/bin/perl -w
  
## Join.pl

use strict; 

my @words = qw{ hello yello mello jello };                         #(A)

my $string = join ':', @words;                                     #(B)
print "$string\n";                  # hello:yello:mello:jello      #(C)

$string = join ' ', @words;         # hello yello mello jello      #(D)
print "$string\n";                                                 #(E)

File: projects/demos/engineering.purdue.edu/scriptingwithobjects/swocode/chap4/Join.py
#!/usr/local/bin/python
  
## Join.py

words = ["hello", "yello", "mello", "jello"]                       #(A)

string = ":".join( words )                                         #(B)
print string                          # hello:yello:mello:jello    #(C)

string = " ".join( words )                                         #(D)
print string                          # hello yello mello jello    #(E)
